doc/cephfs/fs-volumes.rst

```diff
@@ -573,12 +573,27 @@ The commands being deprecated are:
 
 .. note:: Use the ``subvolume info`` command to fetch subvolume metadata regarding supported ``features`` to help decide if protect/unprotect of snapshots is required, based on the availability of the ``snapshot-autoprotect`` feature.
 
+Configure the snapshot_clone_no_wait option :
+
+.. prompt:: bash #
+
+   ``snapshot_clone_no_wait`` config option is used to reject the clone creation request when the cloner threads are not available.
```

Comment:
I think it's good to mention that the default number cloner threads are 4. The number of cloner threads can be configured by ``max_concurrent_clones``

Reply:
Its already mentioned here: https://github.com/ceph/ceph/blob/main/doc/cephfs/fs-volumes.rst?plain=1#L600

Reply:
I think what @kotreshhr meant was to refer that config line from here.